### DISCUSSION TITLE
Have you seen

### DISCUSSION THREAD

**Discussion by Chompalot** [makerforgetech/modular-biped#60] [General]
https://youtu.be/-cfIm06tcfA?si=mbdfQi-BF24HXCDo

**Comment by danic85** [comment]
Oh yes. I have a post about it with all the details available: https://www.makerforge.tech/posts/disney-research/

Maybe we can invite their team to come help us out! 😂


________________________________
From: Simon Cook ***@***.***>
Sent: Monday, October 16, 2023 7:10:46 PM
To: dmt-labs/modular-biped ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [dmt-labs/modular-biped] Have you seen (Discussion #60)


https://youtu.be/-cfIm06tcfA?si=mbdfQi-BF24HXCDo
